### PR TITLE
bugfix: support fact os.name VirtuozzoLinux

### PR DESCRIPTION
**Pull request by benibr**
This is a fix for https://github.com/voxpupuli/puppet-bareos/issues/47

### CODE REVIEW

**Comment by benibr** [comment]
Any feedback or blocker here?

**Comment by benibr** [comment]
Can this be merged please? Right now this blocks a lot of work for us and I do not want to fumble around the locally.

File: manifests/repository.pp [hunk 1]
```diff
@@ -43,6 +43,9 @@
           'Fedora': {
             $location = "${url}Fedora_${osmajrelease}"
           }
+          'VirtuozzoLinux': {
```
Comment (traylenator) [inline manifests/repository.pp]: Given the URL is same as 'RedHat' these two cases can be merged as `'RedHat', 'VirtuozzoLinux'`

Comment (benibr) [reply manifests/repository.pp]: cool. thanks. didn't know that is possible and didn't want to regex. I pushed a new patch

**Comment by benibr** [comment]
Tested and works

**Pull request merged**